src/Util.java

```diff
@@ -0,0 +1,56 @@
+import java.util.Scanner;
+public class Util {
+    final public static Scanner scanner = new Scanner(System.in);
```

Comment:
first of all the order should be:
Access modifier -> static -> final
public/protected/private static final

Second of all:
Why not private?